docs/tracked-extension-resources-are-not-allowed.md

```diff
@@ -0,0 +1,25 @@
+# trackedExtensionResourcesAreNotAllowed
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Uri-V1-12
+
+## Output Message
+
+{0} is an extension resource and the response schema in {1} operation includes location property. Extension resources of type tracked are not allowed.
+
```

Comment:
Can u add an item to standardize the format of all the md files? Some of them seem to have the output message while others may not be having this.

Reply:
[https://msazure.visualstudio.com/One/_workitems/edit/25183186](https://msazure.visualstudio.com/One/_workitems/edit/25183186)